examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    get_booking('./data')
</pre><p> </p><p>do you mind to give the absolute path like <code>/workspace/data</code> instead of <code>.data</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Interestingly, going <code>/workspace/data</code> throws an error. I think there might be an issue in the way getting the data is implemented. Let me check with Marc </p>